manifests/repo/rhel.pp

```diff
@@ -4,20 +4,32 @@
 class rabbitmq::repo::rhel (
   $location          = "https://packagecloud.io/rabbitmq/rabbitmq-server/el/${facts['os'][release][major]}/\$basearch",
   String $key_source = $rabbitmq::package_gpg_key,
+  String $rabbitmq_key_source = 'https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc',
```

Comment:
nit, but looking in other classes, I _think_ these are usually lined up (though it seems like CI is happy).

Reply:
@wyardley i will make the variable name changes was just trying to keep the same as possible to avoid confusion. 

I do not have a way to test Debian.